.github/workflows/k8s_matrix_test.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: K8s version test
+
+on:
+  schedule:
+    # Runs weekly on sundays at noon UTC (I guess)
+    - cron:  '0 12 * * 6'
+  workflow_dispatch:
+
+jobs:
+  run-integration-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: ["v1.17.17", "v1.18.19", "v1.19.11", "v1.20.7", "v1.21.2"]
```

Comment:
`v1.21.2` doesn't yet exist for Kind, we'll need to bump back to `v1.21.1`:

![image](https://user-images.githubusercontent.com/7901615/123636224-98c07180-d7ea-11eb-8bda-63952308cad3.png)